### ISSUE TITLE
Accessing estimated time left within inherited ProgressBarLogger

### ISSUE THREAD

**Issue by yaljazairi** [Edinburgh-Genome-Foundry/Proglog#3]
I was using `MoviePy` to concatenate some videos with each other and reached the point when I must get the percentage values of the progress bar. After searching for a solution I concluded that it was using the `default_bar_logger` in order to show Rendering process in bars view.

For this reason I inherited the `ProgressBarLogger` in a Python class and could until now only get the percentage value of rendering.
The question is: How could we get the value of the estimated time left until rendering is finished (which comes between [ ]. e.g: `02:05<05:00` )? 

```shell 

| 68/100 [00:05<00:02, 12.55it/s, now=None]
```

Here's the implemented logger:

```python

from proglog import ProgressBarLogger

class MyLogger(ProgressBarLogger):
    def callback(self, **changes):
        for (parameter, value) in changes.items():
            print (f"{value}")

    def bars_callback(self, bar, attr, value, old_value=None):
        percentage = (value / self.bars[bar]['total']) * 100
        print(f"percentage: {int(percentage)}")
```

Output:

```shell

percentage: 1
percentage: 2
percentage: 3
.
.
percentage: 100
```

For now I'm only able to get the percentage value correctly but have no idea how to reach estimated time left!

I would appreciate your help for this matter. Thanks in advance!

**Comment by Zulko** [comment]
> How could we get the value of the estimated time left until rendering is finished

I don't believe Proglog enables this for now, two possible ways to solve this would be

- 1) check if there are some `tqdm` attributes from which we can read the bar's estimated remaining time, then extend the `TqdmProgressBar` (with a subclass on your side, or with an MR). Not sure how feasible this is.
- 2)  Since you are creating a custom progress bar class, the progress bar object could compute the estimated remaining time itself, for instance with something like this (top off my head and using variables that don't all )

```python
velocity = loops_since_last_iterations / duration_since_last_iteration
estimated_time = number_of_iterations_left / velocity
```
